install-dev-dependencies.sh

```diff
@@ -65,7 +85,7 @@ elif type dnf >/dev/null ; then
   if [ ! -f /var/lib/dbus/machine-id ]; then
     $SUDO_SHIM sh -c 'dbus-uuidgen > /var/lib/dbus/machine-id'
   fi
-  export PYCURL_SSL_LIBRARY=nss
```

Comment:
nss used to be the default fedora libcurl library, no longer

Reply:
In that case, also consider removing the NSS cipherlist at https://github.com/EFForg/https-everywhere/blob/e2f93d02fe08ef461c1d7260c9290eedbcd63907/test/rules/src/https_everywhere_checker/http_client.py#L48-L49

Reply:
We should update the OpenSSL ciphers as well, since they are currently pinned at those offered by Firefox 51.

Reply:
Probably a task for another day, though.

Reply:
We should also ensure that the macOS development environment doesn't use NSS.

Reply:
I added that `_DEFAULT_CIPHERLIST_NSS` line specifically when I was working on Fedora last year. It seems unlikely that macOS needs it. I understand wanting to test it, though.

Reply:
Note that this also doesn't solve all the problems with Fedora, I'm still running into errors when I try to run the test in docker. This should at least get Fedora systems to a point where the dependencies are installed, though.